crypto/aes/build.info

```diff
@@ -38,6 +38,8 @@ GENERATE[aesv8-armx.S]=asm/aesv8-armx.pl $(PERLASM_SCHEME)
 INCLUDE[aesv8-armx.o]=..
 GENERATE[vpaes-armv8.S]=asm/vpaes-armv8.pl $(PERLASM_SCHEME)
 
+GENERATE[aes-armv4.S]=asm/aes-armv4.pl $(PERLASM_SCHEME)
+GENERATE[bsaes-armv7.S]=asm/bsaes-armv7.pl $(PERLASM_SCHEME)
```

Comment:
Yes, I've rebased against the latest master as you can see the commit history of this PR branch in https://github.com/shigeki/openssl/commits/fix2_raspi2. Your commit of `INCLUDE[aes-armv4.o]=..` at  https://github.com/shigeki/openssl/commit/603358de576217812cb3d752e97c78e476cdc879 which  was landed about 11 hours ago is already included. 

I thought `generatesrc` had an issue but it was not.  My build error log above shows there is no `-P` option which is defined in `generatesrc`. I cannot figure out where this rule comes from.